doc/accelerate_kokkos.txt

```diff
@@ -334,26 +366,30 @@ hardware.
 [Advanced Kokkos options:]
 
 There are other allowed options when building with the KOKKOS package.
-As above, they can be set either as variables on the make command line
-or in Makefile.machine. This is the full list of options, including
-those discussed above. Each takes a value shown below. The default
-value is listed, which is set in the /lib/kokkos/Makefile.kokkos file.
-
-KOKKOS_DEVICES, values = {Serial}, {OpenMP}, {Pthreads}, {Cuda}, default = {OpenMP}
-KOKKOS_ARCH, values = {KNC}, {SNB}, {HSW}, {Kepler30}, {Kepler32}, {Kepler35}, {Kepler37}, {Maxwell50}, {Maxwell52}, {Maxwell53}, {Pascal60}, {Pascal61}, {ARMv80}, {ARMv81}, {ARMv81}, {ARMv8-ThunderX}, {BGQ}, {Power7}, {Power8}, {Power9}, {KNL}, {BDW}, {SKX}, default = {none}
-KOKKOS_DEBUG, values = {yes}, {no}, default = {no}
-KOKKOS_USE_TPLS, values = {hwloc}, {librt}, {experimental_memkind}, default = {none}
-KOKKOS_CXX_STANDARD, values = {c++11}, {c++1z}, default = {c++11}
-KOKKOS_OPTIONS, values = {aggressive_vectorization}, {disable_profiling}, default = {none}
-KOKKOS_CUDA_OPTIONS, values = {force_uvm}, {use_ldg}, {rdc}, {enable_lambda}, default = {enable_lambda} :ul
-
-KOKKOS_DEVICES sets the parallelization method used for Kokkos code
-(within SPARTA). KOKKOS_DEVICES=Serial means that no threading will be
-used.  KOKKOS_DEVICES=OpenMP means that OpenMP threading will be
-used. KOKKOS_DEVICES=Pthreads means that pthreads will be used.
-KOKKOS_DEVICES=Cuda means an NVIDIA GPU running CUDA will be used.
-
-KOKKOS_ARCH enables compiler switches needed when compiling for a
+As above, they can be set either as variables on the make command line,
+in Makefile.machine, on the cmake command line, or in machine.cmake.
+This is the full list of options, including those discussed above.
+Each takes a value shown below. These options are set in the the
+/lib/kokkos/Makefile.kokkos or the kokkos.cmake file. For default values,
+see cmake -LH | grep -i kokkos.
+
+For all Kokkos_* options, values = {ON}, {OFF}
+
+Kokkos_ENABLE_{TYPE}, TYPE = {SERIAL}, {OPENMP}, {PTHREADS}, {CUDA}
+Kokkos_ARCH_{TYPE}, TYPE = {KNC}, {SNB}, {HSW}, {KEPLER30}, {KEPLER32}, {KEPLER35}, {KEPLER37}, {MAXWELL50}, {MAXWELL52}, {MAXWELL53}, {PASCAL60}, {PASCAL61}, {ARMV80}, {ARMV81}, {ARMV81}, {ARMV8-THUNDERX}, {ARMV8-THUNDERX2} {BGQ}, {POWER7}, {POWER8}, {POWER9}, {KNL}, {BDW}, {SKX}, default = {none}
```

Comment:
Unfortunately the all-caps version of KOKKOS_ARCH only works for the CMake build system. Regular gmake wants case-sensitive. I'm trying to get the Kokkos developers to change this in their Makefile.kokkos.